packages/millicast-sdk/src/PeerConnection.js

```diff
@@ -336,11 +336,11 @@ export default class PeerConnection extends EventEmitter {
    *   console.log('Stats from event: ', stats)
    * })
    */
-  initStats () {
+  initStats (autoInitStats, statsIntervalMs) {
     if (this.peerConnectionStats) {
       logger.warn('PeerConnection.initStats() has already been called.  Automatic initialization occurs via View.connect(), Publish.connect() or this.createRTCPeer(). See options')
     } else if (this.peer) {
-      this.peerConnectionStats = new PeerConnectionStats(this.peer)
+      this.peerConnectionStats = new PeerConnectionStats(this.peer, { autoInitStats, statsIntervalMs })
```

Comment:
`options` as well, and in the constructor extract it

Reply:
Si senor